website/app/components/doc/copy-button/index.hbs

```diff
@@ -0,0 +1,14 @@
+<CopyButton
+  class={{this.classNames}}
+  @text={{this.textToCopy}}
+  @onSuccess={{this.onSuccess}}
+  @onError={{this.onError}}
+>
+  {{#if this.textToShow}}
+    <span class="doc-copy-button__visible-value">{{this.textToShow}}</span>
```

Comment:
I imagine we call this `visible-value` to distinguish between what's copied and what's shown

Reply:
Yep! I imagine cases (eg a long text to copy / a long code snippet) in which we want to differentiate what the user sees and what they actually copy.